ssl/ssl_x509.cc

```diff
@@ -981,6 +981,133 @@ int SSL_add1_chain_cert(SSL *ssl, X509 *x509) {
   return ssl_cert_add1_chain_cert(ssl->config->cert.get(), x509);
 }
 
+#define is_flag_set(flags, query) (flags & query)
+
+static int ssl_build_cert_chain(CERT *cert, X509_STORE *cert_store, int flags) {
```

Comment:
Is it possible for cert or cert_store to be null?

Reply:
Judging from the other places in the code where `cert_store` is used directly, it doesn't seem to be possible. But I'll add an `assert` just to be safe.